frontend/src/views/OfferView.vue

```diff
@@ -79,7 +79,29 @@ async function onSaveOffer() {
 
   router.push(`${ROUTES.ORDER.path}/${route.params.order_id}/edit`);
 }
+function onItemDelete(item: OfferItemCreate) {
+  offerItemsArray.value = offerItemsArray.value.filter((element) => element.id !== item.id);
+}
+const allItemsSum = computed(() => {
+  let resultNetto = 0;
+  let resultBrutto = 0;
+  let resultUst = 0;
+
+  for (let i = 0; i < offerItemsArray.value.length; i++) {
+    const amount = offerItemsArray.value[i].amount ?? 0;
+    const price = offerItemsArray.value[i].price ?? 0;
 
+    resultNetto += amount * price;
```

Comment:
Please take a look at the calculation here: https://github.com/fr-ser/scaffolding-management/blob/a8385e629f0187cd7c2521f85d95c0387823df1a/backend/src/pdf/renderPDF.ts#L431

It is important to round all the values to two places as otherwise sub-cent errors pop up in the totals